src/components/info/content/Institution.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+	import type { Content } from "src/api/types";
+
+	export let content: Content;
+	$: names = content.name.split(",");
+	$: json = content.content as any;
+</script>
+
+<div class="mb-5">
+	<h1>{names[0]}</h1>
+
+	{#if names.length > 1}
+	<small>
+		<h6>Also known as:</h6>
```

Comment:
For accessibility reasons, we shouldn't jump from h1 to h6: if this is the next level of heading, let's call it h2 and resize h2 within this scope (e.g. `small h2`) to the desired size. Alternately, we could replace `<small>` with something more semantic like `<header>` and just have the "Also known as:" text be `<strong>` within it. Or the combination: `<header><h2>`.